openpype/hosts/max/api/plugin.py

```diff
@@ -185,7 +185,10 @@ def create_instance_node(node):
             node = rt.Container(name=node)
 
         attrs = rt.Execute(MS_CUSTOM_ATTRIB)
-        rt.custAttributes.add(node.baseObject, attrs)
+        modifier = rt.EmptyModifier()
+        rt.addModifier(node, modifier)
+        node.modifiers[0].name = "OP Data"
```

Comment:
I have question regarging this. Could be `node.modifiers[0]` replaced by `modified`? I do expect it is the same object, but accessing it through index is 1. strange, 2. potential danger if there is different modifier.